From 03ed790fa54961661bd61bb0ab30164eca7cbe57 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 6 Sep 2024 11:46:34 +0800
Subject: [PATCH] [Gentoo] hack(CMakeLists.txt): ensure that Qt 5 is used

We do not yet ship KF6, which is required if a Qt 6 build is specified. The
build system seems to have a broken fallback logic for specifying Qt 5.
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ddb7afcf..1c86ebe3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -272,9 +272,9 @@ if(_qtDir)
   endif()
 endif()
 if(BUILD_WITH_QT6)
-  find_package(QT NAMES Qt6 COMPONENTS Core REQUIRED)
+  find_package(QT NAMES COMPONENTS Core REQUIRED)
 else()
-  find_package(QT NAMES Qt5 Qt6 COMPONENTS Core REQUIRED)
+  find_package(QT NAMES Qt5 COMPONENTS Core REQUIRED)
 endif()
 if(QT_VERSION_MAJOR EQUAL 6 AND QT_VERSION_MINOR LESS 2)
   set(WITH_MULTIMEDIA OFF)
-- 
2.46.0

